docs/next/modules/en/pages/tutorials/quickstart.adoc

```diff
@@ -1,15 +1,69 @@
-= Via Rancher Dashboard
+= Install
 
+{product_name} is a Rancher extension which means that you must have Rancher installed before you can get started with {product_name}.
 
-This is the recommended option for installing {product_name}.
+[TIP]
+====
+If you don't have Rancher installed yet, make sure to follow one of the official https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade[installation guides].
```

Comment:
thinking bit more now, I don't think we need to leave the user a choice of installing Rancher version they want to, because we ship Turtles with specific version of Rancher tested, so exact instructions with specific version of Rancher might need to be still kept as before